### PR TITLE
fix: wrap the footer layout around history page

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Changes

Screenshot 2024-04-03 at 19 20 21

## Events

N / A

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

ref MI-266 

### Preview domain
https://mi-266-footer-nav-missing-on-history.preview.app.daily.dev

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]

File: packages/webapp/pages/history.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { ButtonSize } from '@dailydotdev/shared/src/components/buttons/common';
 import { Dropdown } from '@dailydotdev/shared/src/components/fields/Dropdown';
 import { HistoryType, ReadingHistory } from '../components/history';
 import ProtectedPage from '../components/ProtectedPage';
-import { getLayout } from '../components/layouts/MainLayout';
+import { geHistoryLayout } from '../components/layouts/HistoryLayout';
```
Comment (rebelchris) [inline packages/webapp/pages/history.tsx:24]: You probably took this from `SearchLayout` but feels a bit odd having the same component that doesn't introduce anything new.

I would maybe solve it the same way we do for `getDevCardLayout` or `getNotificationsLayout`
Just avoids having yet another wrapper.

(technically I think we should just remove the SearchLayout, but that's for another ticket.

**Review by rebelchris — CHANGES_REQUESTED** [review]
One small request to avoid a extra file 🙏

**Comment by DragosIorgulescu** [comment]
@rebelchris this is pending your approval to merge 🙏

**Review by rebelchris — APPROVED** [review]
Oh sorry for the delay all good!

**Pull request merged**